### DISCUSSION TITLE
Flye polishing vs Medaka

### DISCUSSION THREAD

**Discussion by oskesr** [mikolmogorov/Flye#696] [Q&A]
Hi all, so we're assembling a relatively large genome using ONT data, and minimap is getting into trouble.
Medaka now has GPU support, and polishing scales up incredibly compared to CPU polishing.

So is there any harm in skipping Flye polishing and heading straight to Medaka ?

**Comment by mikolmogorov** [comment]
If minimap2 is the bottleneck - don't you still need to run it for Medaka?

With respect to the polishing quality - this never been evaluated systematically, I think. To the best of my knowledge Medaka models were trained on Flye-polished assemblies. In addition, flye polisher can correct larger strucural misassmeblies (e.g. 10kb+ indels), and I don't think Medaka can do that. Overall, I'd recommend to do Flye polishing, before Medaka.

**Reply by oskesr** [reply]
Thanks for the info, thats good to know !
Indeed it would've been a problem for Medaka. Turns out it was a hardware issue, I doubled the memory but also went from dual channel to octa-channel. Not sure which of it helped the system, but the assembly finished within 24hrs.
